code/controllers/subsystem/research.dm

```diff
@@ -64,7 +64,7 @@ SUBSYSTEM_DEF(research)
 				bitcoins[i] = bitcoins[i]? bitcoins[i] + result[i] : result[i]
 	else
 		for(var/obj/machinery/rnd/server/miner in servers)
-			if(miner.working)
+			if(!miner.machine_stat)
 				bitcoins = single_server_income.Copy()
 				break			//Just need one to work.
```

Comment:
`!miner.machine_stat` is a bit difficult to read. I think this will be better:
```DM
		for(var/obj/machinery/rnd/server/miner in servers)
			if(miner.machine_stat) // unable to work
				continue
			bitcoins = single_server_income.Copy()
			break	//Just need one to work.
```
I am asking you to use `continue` instead of using `not` operator.

Reply:
done